hermes/templates/backup-secret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "hermes.name" . }}-store-backup
+  labels:
+    app: {{ template "hermes.name" . }}-store-backup
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  hermes-store-backup.aws-access-key-id: {{ .Values.backup.awsaccesskey }}
```

Comment:
Remove this prefix `hermes-store-backup` - its obsolete and just make lines long and less readable